ci/github-actions.md

```diff
@@ -0,0 +1,125 @@
+
+Next, define a file called `main.yml` in the workflows directory, and paste in the following configuration:
+
+```yaml
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build containers and run tests
+      run: docker-compose -f docker-compose.yml -f tests/docker-compose.yml run --rm app
+```
+
+Commit this file to your feature branch and open up a pull request. You should be able to confirm that your workflow runs the tests for your pull request.
+
+### Examples
+
+This workflow is in use in the following projects:
+
+- [Minnesota Election Archive](https://github.com/datamade/mn-election-archive) (visible to DataMade developers only)
+
+## Deploying a Python package
+
+GitHub Actions can run arbitrary code, so we can use it to deploy Python packages to PyPI in addition to testing them.
+
+As above, start by making a new feature branch and creating the directory `.github/workflows/` at the root of your repo if it doesn't yet exist. Then, define two workflow files, one for each of the production and test PyPI instances:
+
+#### 📄`.github/workflows/publish-to-pypi.yml`
+
+```yaml
+name: Publish to PyPI
+
+on: push
+
+jobs:
+  build-and-publish:
+    name: Publish to PyPI
+    if: startsWith(github.event.ref, 'refs/tags')
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Publish to PyPI
+      env:
+        TWINE_USERNAME: '__token__'
+        TWINE_PASSWORD: ${{ secrets.pypi_password }}
+      run: |
+        pip install twine wheel
+        pip wheel -w dist --no-deps .
+        twine upload --skip-existing dist/*
+      continue-on-error: true
+```
+
+#### 📄`.github/workflows/publish-to-test-pypi.yml`
+
+```yaml
+name: Publish to Test PyPI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-publish:
+    name: Publish to Test PyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Publish to Test PyPI
+      env:
+        TWINE_USERNAME: '__token__'
+        TWINE_PASSWORD: ${{ secrets.test_pypi_password }}
+      run: |
+        pip install twine wheel
+        pip wheel -w dist --no-deps .
+        twine upload --skip-existing --repository-url https://test.pypi.org/legacy/ dist/*
+      continue-on-error: true
+```
+
+Next, follow the instructions for [creating encrypted secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets#creating-encrypted-secrets) and save values for `pypi_password` and `test_pypi_password`.
```

Comment:
These should be API keys, not account passwords. I was following directions from [here](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#saving-credentials-on-github). Now that I'm thinking about it these names are confusing! They could be changed to `pypi_api_token` and `test_pypi_api_token`

Reply:
Ah, good catch. I changed the names of the variables in this file and also added a link to the official Python docs to the `Resources` section.